### PR TITLE
BAQE-336: Cover the plain drl service

### PR DESCRIPTION
**Pull request by jomarko**
This PR cover basic operation of the plain drl service like save, create, load, validate.

@manstis please have a look.

### CODE REVIEW

File: drools-wb-screens/drools-wb-drl-text-editor/drools-wb-drl-text-editor-backend/src/test/resources/org/drools/workbench/screens/drltext/backend/server/drl/pom.xml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.kiegroup</groupId>
+  <artifactId>drl</artifactId>
+  <version>1.0</version>
+  <packaging>kjar</packaging>
+  <name>drl</name>
+  <dependencies>
+    <dependency>
```
Comment (manstis) [inline drools-wb-screens/drools-wb-drl-text-editor/drools-wb-drl-text-editor-backend/src/test/resources/org/drools/workbench/screens/drltext/backend/server/drl/pom.xml]: I doubt we need this (I know, I know.. the project was _probably_ created in Business Central).

File: drools-wb-screens/drools-wb-drl-text-editor/drools-wb-drl-text-editor-backend/src/test/resources/org/drools/workbench/screens/drltext/backend/server/drl/pom.xml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      <groupId>org.jboss.qa.droolsjbpm</groupId>
+      <artifactId>workitems</artifactId>
+      <version>1.0.0-FINAL</version>
+    </dependency>
+  </dependencies>
+  <build>
+    <plugins>
+      <plugin>
```
Comment (manstis) [inline drools-wb-screens/drools-wb-drl-text-editor/drools-wb-drl-text-editor-backend/src/test/resources/org/drools/workbench/screens/drltext/backend/server/drl/pom.xml]: I doubt we need this... and, if we do, I doubt we want to use ```6.2.0.CR3-redhat-1```

File: drools-wb-screens/drools-wb-drl-text-editor/drools-wb-drl-text-editor-backend/src/test/resources/org/drools/workbench/screens/drltext/backend/server/drl/src/main/resources/META-INF/kie-deployment-descriptor.xml [hunk 3]
```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```
Comment (manstis) [inline drools-wb-screens/drools-wb-drl-text-editor/drools-wb-drl-text-editor-backend/src/test/resources/org/drools/workbench/screens/drltext/backend/server/drl/src/main/resources/META-INF/kie-deployment-descriptor.xml]: We should not need this file.

File: drools-wb-screens/drools-wb-drl-text-editor/drools-wb-drl-text-editor-backend/src/test/resources/org/drools/workbench/screens/drltext/backend/server/drl/src/main/resources/META-INF/kmodule.xml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+<kmodule xmlns="http://jboss.org/kie/6.0.0/kmodule" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <kbase name="kbase1" default="true" eventProcessingMode="stream" equalsBehavior="identity">
+    <ksession name="ksession" type="stateful" default="true" clockType="realtime">
+      <workItemHandlers>
```
Comment (manstis) [inline drools-wb-screens/drools-wb-drl-text-editor/drools-wb-drl-text-editor-backend/src/test/resources/org/drools/workbench/screens/drltext/backend/server/drl/src/main/resources/META-INF/kmodule.xml]: Can you remove the WIH definitions?

**Comment by jomarko** [comment]
@manstis I cleaned up all CDI test resources, would you mind to re-review please?

**Pull request merged**